### PR TITLE
Favorites hover color

### PR DESCRIPTION
**Pull request by leo-fsl**

### CODE REVIEW

File: src/components/Cards/shared/Title/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const TitleTheme = {
 
     @media(hover: hover) {
       &:hover {
-        color: ${color.mint};
+        color: ${color.grayishCyan};
```
Comment (jwinnfeild2517) [inline src/components/Cards/shared/Title/index.js:13]: I would not change the shared component here as it will also be changed in other areas of the site.

**Review by jwinnfeild2517 — COMMENTED** [review]
The madeForyou card also needs an HoverState According to the card

File: src/components/Cards/shared/Title/index.js (hunk 1 above)
Comment (leo-fsl) [reply src/components/Cards/shared/Title/index.js]: Removed change

File: src/components/Cards/MadeForYouCard/index.js [hunk 2]
```diff
@@ -16,6 +16,12 @@ const MadeForYouCardWrapper = styled.a`
     height: 230px;
     display: block;
   }
+
+  &:hover {
+    h3 {
+      color: ${color.grayishCyan};
```
Comment (jwinnfeild2517) [inline src/components/Cards/MadeForYouCard/index.js:22]: I think the entire card also has to shift up a bit

File: src/components/Cards/RelatedRecipeCard/RelatedRecipeCard.tsx [hunk 3]
```diff
@@ -58,6 +53,10 @@ const Headline = styled.span`
   ${withThemes(HeadlineTheme)}
   ${cssThemedColor}
   ${mixins.truncateLineClamp(3)}
+
```
Comment (jwinnfeild2517) [inline src/components/Cards/RelatedRecipeCard/RelatedRecipeCard.tsx:56]: place the hover state on the entire card. So when the user hovers on the card. the title changes color

File: src/components/Cards/MadeForYouCard/index.js (hunk 2 above)
Comment (leo-fsl) [reply src/components/Cards/MadeForYouCard/index.js]: Done

File: src/components/Cards/RelatedRecipeCard/RelatedRecipeCard.tsx (hunk 3 above)
Comment (leo-fsl) [reply src/components/Cards/RelatedRecipeCard/RelatedRecipeCard.tsx:56]: As said, the hover is only on the title

**Pull request merged**